omniNotes/src/androidTest/java/it/feio/android/omninotes/utils/MoreOptions.java

```diff
@@ -0,0 +1,33 @@
+package it.feio.android.omninotes.utils;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.matcher.ViewMatchers.withContentDescription;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static it.feio.android.omninotes.utils.EspressoUtils.addPasswordWhenRequested;
+import static it.feio.android.omninotes.utils.EspressoUtils.clickOnButton;
+import static it.feio.android.omninotes.utils.NewNoteStringVariables.archiveButton;
+import static it.feio.android.omninotes.utils.NewNoteStringVariables.lockButton;
+import static it.feio.android.omninotes.utils.NewNoteStringVariables.moreOptionsButton;
+import static it.feio.android.omninotes.utils.NewNoteStringVariables.trashButton;
+
+public class MoreOptions {
+    public static void openMoreOptionsMenu(){
+        clickOnButton(onView(withContentDescription(moreOptionsButton)));
+    }
+
+    static void clickOnTrash(){
+        openMoreOptionsMenu();
+        clickOnButton(onView(withText(trashButton)));
+    }
+
+    static void clickOnLock(){
+        openMoreOptionsMenu();
+        clickOnButton(onView(withText(lockButton)));
+        addPasswordWhenRequested();
```

Comment:
this should be done elsewhere, keep the methods doing only one thing

Reply:
this is needed because after you set a password, this password is required anytime you want to lock a note

Reply:
👍